airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -795,6 +799,14 @@ public String getDDDogStatsDPort() {
     return getEnvOrDefault(DD_DOGSTATSD_PORT, "");
   }
 
+  @Override
+  public String getDDPrefix() {
+    String prefix = getEnvOrDefault(DD_PREFIX, "").strip();
+    if (StringUtils.isNotEmpty(prefix) &&  !prefix.endsWith(".")){
```

Comment:
nit: Since we are using StringUtils, [appendIfMissing](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#appendIfMissing-java.lang.String-java.lang.CharSequence-java.lang.CharSequence...-) might be useful here.

Reply:
Thanks for the suggestion!